drivers/bridge/bridge.go

```diff
@@ -603,6 +604,9 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 	}
 
 	setupNetworkIsolationRules := func(config *networkConfiguration, i *bridgeInterface) error {
+		if config.Internal {
+			return nil
+		}
```

Comment:
We also want to avoid any nw1 <-> nw2 isolation rule if either nw1 or nw2 is internal.
So, instead of the above change, please modify `isolcatenetwork()` function to:

```
func (n *bridgeNetwork) isolateNetwork(others []*bridgeNetwork, enable bool) error {
	n.Lock()
	thisConfig := n.config
	n.Unlock()

	if thisConfig.Internal {
		return nil
	}

	// Install the rules to isolate this networks against each of the other networks
	for _, o := range others {
		o.Lock()
		otherConfig := o.config
		o.Unlock()

		if otherConfig.Internal {
			continue
		}

		if otherConfig.BridgeName != otherIface {
			if err := setINC(thisConfig.BridgeName, otherConfig.BridgeName, enable); err != nil {
				return err
			}
		}
	}

	return nil
}
```